RELEASING.md

```diff
@@ -57,19 +60,16 @@ git checkout -b prepare-0.9.0
 # RHEL: rpm/cloud-info-provider.spec
 # RHEL: rpm/cloud-info-provider-opennebula.spec
 # RHEL: rpm/cloud-info-provider-openstack.spec
+# Update Zenodo configuration if needed
 # Update AUTHORS file if needed
 vim AUTHORS
-# Update Zenodo configuration
-# Description, title, version, publication_date, creators
-vim .zenodo.json
 # Commit changes
 git commit -am 'Prepare release 0.9.0'
-# Push new branch to fork
-git push --set-upstream origin prepare-0.9.0
 # Create pull request
-git pull-request
+gh pr create
 # After PR merging
-git release create -m 0.9.0 0.9.0
+git tag -m 0.9.0 0.9.0
+git push --tags
```

Comment:
Instead of using `git push --tags` it may be better/safer to use something like `git push origin <tag_name>`.
And here those two cases are pushing to `origin` so not to `upstream`.
We should be able to do it from a GitHub action, as we should/could have a token allowing to push to the repo.  For this when there is a commit pushed to `master` we would need a way to figure if we need to trigger the release creation, with package building, tag creation and upload of artefacts.

Reply:
Rethinking this... will get back to release create